### PR TITLE
simplify liftingline and unify C_Mx calculation

### PR DESCRIPTION
**Pull request by helo9**
* removes redundant code
* implement new calculation grid determination

### CODE REVIEW

File: wingstructure/aero/liftingline.py [hunk 1]
```diff
@@ -74,30 +81,23 @@ def __init__(self):
...
+            A = self._area
+            b = self._spanwidth
+            C_Mx = np.trapz(self.ys*res.c_ls*self.chords, self.ys)/ (A*b)
 
-        c_ls += α * self._aoa.c_ls
-        C_Di += abs(α) * self._aoa.C_Di
+            return np.rad2deg(α), res.c_ls, res.C_Di, C_Mx
 
-        return np.rad2deg(α), c_ls, C_Di
+        return np.rad2deg(α), res.c_ls
```
Comment (dernatsch) [inline wingstructure/aero/liftingline.py]: Maybe we should keep the api consistent and return None for fields that are not calculated.

File: wingstructure/wingloads.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    dihedral. Flat means parallel to the x-y-plane.
+    
+    Parameters
+    ----------
+    wing : Wing
+        the wing to be flattend
+    """
+    
+    def __init__(self, wing):
+        """create flat wing instance
```
Comment (dernatsch) [inline wingstructure/wingloads.py]: what does this do exactly? :wink:

File: wingstructure/aero/liftingline.py (hunk 1 above)
Comment (helo9) [reply wingstructure/aero/liftingline.py]: Valid point, but I think this behavior is not uncommon in python.

File: wingstructure/wingloads.py (hunk 2 above)
Comment (helo9) [reply wingstructure/wingloads.py]: obvious enough!

**Pull request merged**